spec/components/trainees/record_details/view_spec.rb

```diff
@@ -69,6 +71,38 @@ module RecordDetails
             expect(component.find(summary_card_row_for(trainee_status))).to have_text(date_for_summary_view(trainee.withdraw_date))
           end
         end
+
+        context "when trainee state is recommended_for_award" do
+          let(:state) { :recommended_for_award }
+
+          it "renders the trainee recommended date" do
+            expect(component.find(summary_card_row_for("qts"))).to have_text(date_for_summary_view(trainee.recommended_for_award_at))
```

Comment:
Can we update all the `component.find(summary_card_row_for("qts"))` to `rendered_component` then we can get rid of the alias stuff at the top. The assertions should just work on the rendered output

Reply:
Done!